Gemfile

```diff
@@ -55,6 +55,7 @@ gem "hamlit",                         "~>2.7.0"
 gem "htauth",                         "2.0.0",         :require => false
 gem "inifile",                        "~>3.0",         :require => false
 gem "jbuilder",                       "~>2.5.0" # For the REST API
+gem "kubeclient",                     "~>2.4.0",       :require => false # For scaling pods at runtime
```

Comment:
Should we be this restrictive on this dependency?  Maybe "~>2.4" would be better to avoid conflict with the provider repo? (Assuming they follow semver)